serv/website/sound.html

```diff
@@ -38,6 +38,108 @@
         </nav> <!-- END OF NAVBAR-EXPAND -->
     </header> <!-- END OF HEADER -->
     <main>
+        <div id="content-box" class="container-fluid">
+            <h2 class="text-uppercase">SOUND AROUND YOU<br> <span class="subheadercb">MERSEBURG</span></h2>
+            <div class="row">
+                <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 col-xl-6">
+                    <div class="row row-cols-2"> <!-- row-cols-2 -->
+                        <img src="images/say3.jpeg" class="col img-fluid" alt="Workshop">
```

Comment:
we should provide image descriptions in the `alt` tags, i.e. to help visually impaired people using screenreaders or to help with seo. an exemplary use here could be 
´´`html
<img src="images/say3.jpeg" class="col img-fluid" alt="field recording workshop in merseburg">
```
 you can read up more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt)

Reply:
Done